sdk/test-utils/recorder/MIGRATION.md

```diff
@@ -152,23 +152,28 @@ In this example, the name of the queue used in the recording is randomized. Howe
 
 A powerful feature of the legacy recorder was its `customizationsOnRecordings` option, which allowed for arbitrary replacements to be made to recordings. The new recorder's analog to this is the sanitizer functionality.
 
-### GeneralRegexSanitizer
+### General sanitizers
 
-For a simple find/replace, a `GeneralRegexSanitizer` can be used. For example:
+For a simple find/replace, `generalSanitizers` can be used. For example:
 
 ```ts
 await recorder.addSanitizers({
-  generalRegexSanitizers: [
+  generalSanitizers: [
```

Comment:
There are more "regexSanitizers", right? @timovv 
Would they all have string sanitizer counterparts?

If so, more breaking changes?

/cc @scbedd

Reply:
this seems like a breaking change indeed

Reply:
(but I like it!)

Reply:
Yes, they all have string sanitizer counterparts, all of which I have added support for. The [Sanitizers source directory](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Sanitizers) shows all of them.

Reply:
And yes, breaking changes for all... but since we haven't migrated much over yet, I don't think the impact is significant!

Reply:
as part of this PR, should we include migrations to these changes to anything that has already migrated?

Reply:
Great point. None of the migrated packages that I can see are using this API at the moment, so no changes are necessary!